wework/src/i18n/locales/zh-CN/common.json

```
@@ -239,8 +249,18 @@
+    "project_automation_waiting_approval": "等待批准",
+    "project_automation_starting": "正在启动",
+    "project_automation_waiting_runtime": "等待运行时确认",
     "project_automation_running": "运行中",
+    "project_automation_cancelling": "正在取消",
+    "project_automation_succeeded": "成功",
+    "project_automation_failed": "失败",
+    "project_automation_cancelled": "已取消",
+    "project_automation_skipped": "已跳过",
+    "project_automation_unknown": "状态待核实",
     "project_automation_completed": "已完成",
+    "project_automation_run_details": "查看运行详情",
```

Biome (2.5.6)

[error] The key project_automation_run_details was already declared.

(lint/suspicious/noDuplicateObjectKeys)